pkg/commands/compute/setup/config_store.go

```diff
@@ -48,13 +50,45 @@ type ConfigStoreItem struct {
 
 // Configure prompts the user for specific values related to the service resource.
 func (o *ConfigStores) Configure() error {
+	existingStores, err := o.APIClient.ListConfigStores()
+	if err != nil {
+		return err
+	}
+
 	for name, settings := range o.Setup {
+		var (
+			existingStoreID   string
+			linkExistingStore bool
+		)
+
+		for _, store := range existingStores {
+			if store.Name == name {
+				if o.AcceptDefaults || o.NonInteractive {
+					linkExistingStore = true
+					existingStoreID = store.ID
+				} else {
+					text.Warning(o.Stdout, "\nA Config Store called '%s' already exists\n\n", name)
+					prompt := text.Prompt("Use a different store name (or leave empty to use the existing store): ")
```

Comment:
Is it worth saying here that data will be upserted?

Reply:
That's a fair point actually, as it's only relevant when dealing with an existing store 👍🏻